common/logisticspipes/gui/guidebook/GuiGuideBookSlider.java

```diff
@@ -0,0 +1,87 @@
+package logisticspipes.gui.guidebook;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.audio.SoundHandler;
+import net.minecraft.client.gui.GuiButton;
+import net.minecraft.client.renderer.GlStateManager;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.math.MathHelper;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import logisticspipes.LPConstants;
+
+public class GuiGuideBookSlider extends GuiButton {
+
+	private static final ResourceLocation GUI_BOOK_TEXTURE = new ResourceLocation(LPConstants.LP_MOD_ID, "textures/gui/guide_book.png");
+
+	@Getter
+	private float progress;
+	private boolean dragging;
+	public int max, min;
+
+	public void setProgress(float progress){
```

Comment:
Why is *setProgress* not setting `this.progress`?

Reply:
also code style, spacing

Reply:
Because it was only half made and unused :( But I fixed it and gave it some purpose in life